taxonomy-manager-engine/src/main/java/com/digirati/taxonomy/manager/lookup/TextLookupService.java

```diff
@@ -24,40 +24,42 @@
 /** Service to orchestrate all the steps in looking up concepts from a piece of input text. */
 public class TextLookupService {
 
+    private static final Set<String> stopWords = Sets.newHashSet(
+            "a", "an", "and", "are", "as", "at", "be", "but", "by",
+            "for", "if", "in", "into", "is", "it",
+            "no", "not", "of", "on", "or", "such",
+            "that", "the", "their", "then", "there", "these",
+            "they", "this", "to", "was", "will", "with"
+    );
+
     private final TextNormaliser textNormaliser;
 
     private final ConceptExtractor conceptExtractor;
 
     private final String languageKey;
 
     public static Future<TextLookupService> initialiseLookupService(
-            Stream<ConceptModel> concepts,
+            Stream<? extends Concept> concepts,
```

Comment:
`Concept` can be something like:

```java
class Term {
    Term(UUID id, Set<String> labels) { ... }
}
```

We don't need to know about concepts other than their identities and labels within the engine.